### PR TITLE
Create chinese_remainder_theorem.py

### PR DESCRIPTION
**Pull request by ashu-cybertron**
Add it as my contribution

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
yes I have read.
### Description

(Write your answer here.)
This is code for the Chinese remainder theorem in python.
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
def multiply_fun(li):
    result = 1
    for i in li:
        result *= i
    return result

k = int(input("Enter size of array : "))
print("Enter value of coprime divisors :", end=' ')
n = list(map(int, input() . split()))
print("Enter value of remainders :", end=' ')
a = list(map(int, input().split()))

def chineserem(n, a):
    sum = 0
    pr = multiply_fun(n)
    for n_i, a_i in zip(n, a):
        p = pr/n_i
        sum += a_i * muinv(p, n_i)*p
    return sum % pr

def muinv(a, b):
    b0 = b
    x0, x1 = 0, 1
    if b == 1:
        return 1
    while a > 1:
        q = a//b
        a, b = b, a % b
        x0, x1 = x1 - q * x0, x0
    if x1 < 0:
        x1 += b0
    return x1

print("x = ", chineserem(n, a))

Sample Input -->
Enter size of array : 3
Enter value of coprime divisors : 2 3 5
Enter value of remainders : 1 2 4

Output -->
x =  29.0

it is the program of the Chinese remainder system in python please add it as my contribution

### CODE REVIEW

**Comment by ashu-cybertron** [comment]
please check

**Review by iamrajiv — COMMENTED** [review]
@ashu-cybertron Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

File: Python/other/chinese_remainder_theorem.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from functools import reduce
```
Comment (atarax665) [inline Python/other/chinese_remainder_theorem.py]: Do not use built in functions.

File: Python/other/chinese_remainder_theorem.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        q=a// b
+        a, b= b, a%b
+        x0, x1=x1 -q *x0, x0
+    if x1<0 : x1+= b0
+    return x1
+#n=[2,3,5]
+#a=[1,2,4]
+
+    
+print("x=",chineserem(n,a))
```
Comment (atarax665) [inline Python/other/chinese_remainder_theorem.py]: Add Sample Input/Output and Time Complexity of your code at the end in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Review by ashu-cybertron — COMMENTED** [review]
please check the changes @atarax665 @iamrajiv

File: Python/other/chinese_remainder_theorem.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+def mulitply(li):
```
Comment (iamrajiv) [inline Python/other/chinese_remainder_theorem.py]: Fix typo mistake

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ashu-cybertron Update the Readme.

**Review by ashu-cybertron — COMMENTED** [review]
it is improved see this @atarax665 @iamrajiv

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Comment by vatsalkesarwani12** [comment]
update readme.md file

**Comment by ashu-cybertron** [comment]
i have updated readme @atarax665 @iamrajiv  plz merge it and make me assignee

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]

**Comment by atarax665** [comment]
@all-contributors add @ashu-cybertron  for code :+1:

**Pull request merged**